lib/ai.py

```diff
@@ -23,27 +23,31 @@ def __init__(self, temperature=0.1):
 
     def print(self, *args):
         return self.printer.print(*args)
+    
 
-    def next(self, desc: str, page_size=10) -> List[str]:
+    
+
+    def next(self, desc: str, page_size: Optional[str] = 10,tlds: Optional[str]= '.com' '.ai' '.io') -> List[str]:
         prompt = ''
         with open("prompts/default", "r") as file:
             prompt = file.read()
             prompt = prompt.replace("{page_size}", str(page_size))
             prompt = prompt.replace("{desc}", desc)
-            prompt = prompt.replace("{tlds}", ', '.join(VALID_TLDS))
+            prompt = prompt.replace("{tlds}", (tlds))
```

Comment:
When you turn `tlds` into a list of strings please join them with comma like so 
```suggestion
            prompt = prompt.replace("{tlds}", (','.join(tlds)))
```